### PR TITLE
feat(PresenceBadge) Add `blocked` status

### LINKED ISSUE #26362: PresenceBadge should have blocked status
The icon for this presence status already exists as `PresenceBlocked`. You can search for that icon in the docsite https://react.fluentui.dev/?path=/docs/concepts-developer-icons-icons-catalog--page.

This is a partner ask by Teams, since there is a feature to block a user

### PR DESCRIPTION
**Pull request by jurokapsiar**
## Previous Behavior

PresenceBadge was missing blocked status

## New Behavior

PresenceBadge has blocked status and Avatar can use it:
image

Spec: 
image

## Related Issue(s)

* Fixes #26362

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4cf244f9b08632154079a815c62c3b96ac2f1310'>4cf244f9b08632154079a815c62c3b96ac2f1310</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Avatar-mount_base.html">1307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Avatar-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Button-mount_base.html">930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Button-mount.html">862</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProvider-mount_base.html">1548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProvider-mount.html">1595</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-mount_base.html">626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-mount.html">630</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">585</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">628</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/InfoButton-mount_base.html">566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/InfoButton-mount.html">533</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/MakeStyles-mount_base.html">1905</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/MakeStyles-mount.html">1899</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Persona-mount_base.html">2992</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/Persona-mount.html">2985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/SpinButton-mount_base.html">2500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26366/perf-test-react-components/SpinButton-mount.html">2541</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {
   presenceAvailableRegular,
   presenceAwayFilled,
   presenceBusyFilled,
+  presenceBlockedRegular,
```
Comment (khmakoto) [inline packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx:8]: _nit:_ Keep alphabetical order

```suggestion
  presenceBlockedRegular,
  presenceBusyFilled,
```

**Review by khmakoto — APPROVED** [review]
We should probably ask design to update the `Presence table` in the spec to add the `blocked` status.

image

File: change/@fluentui-react-badge-004df5ab-74c6-4945-9b7f-acd5e6916482.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-badge-004df5ab-74c6-4945-9b7f-acd5e6916482.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26366&commitId=0dd03eb398ec9d1c6ddacd0f40e4326581c654f6&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1769 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|395| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|597| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|602| Changed|
|Avatar Converged.basic.normal.chromium.png|615| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Pull request merged**